common/models/models.go

```diff
@@ -141,3 +144,15 @@ type StreamSettings struct {
 	ScalingFactorAndroid int `json:"scaling_factor_android,omitempty" bson:"scaling_factor_android"`
 	ScalingFactoriOS     int `json:"scaling_factor_ios,omitempty" bson:"scaling_factor_ios"`
 }
+
+type Workspace struct {
+	ID          string `json:"id" bson:"_id,omitempty"`
```

Comment:
I am 99% sure that the _id cannot be empty unless explicitly set to empty in some way, I guess `omitempty` is redundant

Reply:
When not using `omitempty`, the value stored in the database is `""` because we are not setting its value, and the serialization is not removing the empty string.

Reply:
Yes, because we are using `string` which is not the idiomatic type for MongoDB documents, it should be `primitive.ObjectID`. Unfortunately I've written some code where I didn't read the docs comprehensively or spent much time thinking on it and we already have some string types for `_id` here and there 😓 🤣